### PR TITLE
docs: Fix OAuth casing

### PR DESCRIPTION
**Pull request by keithwillcode**
## What does this PR do?

Fixes places where we had `oauth` instead of `OAuth`

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [x] I have updated the developer docs in /docs if this PR makes changes that would require a [documentation change](https://cal.com/docs). If N/A, write N/A here and check the checkbox.
- [x] I confirm automated tests are in place that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by Udit-takkar — APPROVED** [review]

File: apps/api/v2/src/ee/calendars/services/gcal.service.ts [hunk 1]
```diff
@@ -4,14 +4,14 @@ import { AppsRepository } from "@/modules/apps/apps.repository";
 import { CredentialsRepository } from "@/modules/credentials/credentials.repository";
 import { SelectedCalendarsRepository } from "@/modules/selected-calendars/selected-calendars.repository";
 import { TokensRepository } from "@/modules/tokens/tokens.repository";
+import { calendar_v3 } from "@googleapis/calendar";
```
Comment (youneshenniwrites) [inline apps/api/v2/src/ee/calendars/services/gcal.service.ts:7]: was this change necessary?

**Review by hbjORbj — APPROVED** [review]

File: apps/api/v2/src/ee/calendars/services/gcal.service.ts (hunk 1 above)
Comment (Ryukemeister) [reply apps/api/v2/src/ee/calendars/services/gcal.service.ts:7]: @youneshenniwrites don't think this will affect anything since its just a formatting change.
